recipe/meta.yaml

```diff
@@ -48,7 +48,6 @@ requirements:
     - tifffile >=2022.8.12
     - pywavelets >=1.1.1
     - packaging >=21
-    - lazy_loader >=0.4
```

Comment:
we already depend on lazy-loader on top. the underscore version is the "old way"